packages/commons/src/fhir/transformers/index.ts

```diff
@@ -2253,75 +2243,45 @@ export const builders: IFieldBuilders = {
       const taskResource = selectOrCreateTaskRefResource(fhirBundle, context)
       taskResource.id = fieldValue as string
     },
-    informantsSignature: async (fhirBundle, fieldValue, context) => {
+    informantsSignature: (fhirBundle, fieldValue, context) => {
       const taskResource = selectOrCreateTaskRefResource(fhirBundle, context)
-      if (isBase64FileString(fieldValue)) {
```

Comment:
Asyncronous operations should all be done before we try to transform data into FHIR. The purpose of this is to move our logic outside FHIR as much as we can and create a pure `OpenCRVS data -> FHIR` transformer.

See [this commit](https://github.com/opencrvs/opencrvs-core/pull/6020/commits/e5f7f2a70197c8585715ce15accbd2412b7271e1) for new program flow

Reply:
The commit seems to have been lost